### DISCUSSION TITLE
Function which takes user input for memcached

### DISCUSSION THREAD

**Discussion by rishabh-pokarna** [memcached/memcached#990] [Q&A]
I was reading through the code in the "memcached.c" file and was trying to find the part where the user input is received once the connection is established between the server and the client?

**Comment by dormando** [comment]
The code isn't going to be super easy to follow; it's multithreaded and event based. The main network code is `drive_machine()` in `memcached.c` - that checks from the network and dispatches commands. The command handlers are in the various proto_* files.
